model_compression_toolkit/core/runner.py

```diff
@@ -45,7 +46,7 @@
     WeightsFinalBitwidthConfigVisualizer, \
     ActivationFinalBitwidthConfigVisualizer
 from model_compression_toolkit.core.common.visualization.tensorboard_writer import TensorboardWriter
-
+from model_compression_toolkit.core.common.hessian import hessian_service
```

Comment:
Following the comment on the hessian_service variable, I think it would be better to initiate it here inside the runner instead of globally outside the service class.